packages/server/lib/browsers/chrome.ts

```diff
@@ -92,6 +92,9 @@ const DEFAULT_ARGS = [
   '--safebrowsing-disable-download-protection',
   '--disable-client-side-phishing-detection',
   '--disable-component-update',
+  // Simulate when chrome needs an update.
+  // This prevents an 'update' from displaying til the given date
+  `--simulate-outdated-no-au='Tue, 31 Dec 2099 23:59:59 GMT'`,
```

Comment:
such tech debt, we're gonna have to deal with this in only 78 years!

/s

on a serious note, i saw a couple other ways to disable this, but i guess they're all equally as weird as this:

```ts
--check-for-update-interval=31536000
--simulate-update-hresult=something, not exactly sure
```